nautobot/docs/installation/tables/v2-code-removals.yaml

```diff
@@ -13,6 +13,8 @@
   Class/Function(s): "ButtonActionColorChoices"
 - Module: "nautobot.core.choices"
   Class/Function(s): "ButtonActionIconChoices"
+- Module: "nautobot.core.filters"
+  Class/Function(s): "NameSlugSearchFilterSet"
```

Comment:
Should this be in docs/installation/tables/v2-code-location-changes.yaml

Reply:
I listed it in removals since the `NameSearchFilterSet` isn't a one-for-one replacement - any app that still wants to have name+slug as their search criteria will need to implement their own search filter.